src/main/java/io/appform/dropwizard/actors/base/UnmanagedPublisher.java

```diff
@@ -234,6 +234,7 @@ private void ensureExchange(String exchange) throws IOException {
     private void ensureDelayedExchange(String exchange) throws IOException {
         if (config.getDelayType() == DelayType.TTL) {
             ensureExchange(ttlExchange(config));
+            ensureExchange(exchange);
```

Comment:
@swapnil-tagwale Add test cases for this or add assertions in existing test cases